tst/EnergyPlus/unit/DesiccantDehumidifiers.unit.cc

```diff
@@ -6523,8 +6523,8 @@ TEST_F(EnergyPlusFixture, DesiccantDehum_VSCoolingCoilOnPrimaryAirSystemTest)
         "    Desiccant DXSystem Cooling Coil,    !- Name",
         "     Desiccant DXSystem Mixed Air Node,  !- Indoor Air Inlet Node Name",
         "    HX Process Inlet Node,   !- Indoor Air Outlet Node Name",
-        "    1.0,                     !- Number of Speeds {dimensionless}",
-        "    1.0,                     !- Nominal Speed Level {dimensionless}",
```

Comment:
I went ahead and changed most of the IDF snippets in unit tests to be the correct type. I believe the only two left are situations in which rounding (1.9 and 2.1 rounded to 2) is important.

Reply:
:+1: